config/dev.webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const { resolve } = require('path');
+const config = require('@redhat-cloud-services/frontend-components-config');
```

Comment:
This is a useful config lib I ran across during my HAC-Core work with ConsoleDot. Their insights projects have made things a lot nicer to use... never enjoyed the ease of working with Webpack more 🙂

Reply:
> the ease of working with Webpack

This made me chuckle since Webpack is easy until it's not.

Reply:
Shhh @rhamilto, it'll hear you!

I kid, this is useful to abstract _some_ use-cases into easier flows. I have felt it was useful to help build proxies and what not, but yeah -- it's not exactly an easy-button solution. "easy" -- with quotes 🙂